src/actions/google/ads/lib/ads_request.ts

```diff
@@ -122,17 +127,23 @@ export class GoogleAdsActionRequest {
   async execute() {
     // 0) Do execution specific validations
     if (!this.loginCid) {
+      winston.warn(`Login account id is missing, webhookID=${this.webhookId}`)
```

Comment:
winston.warn(`Login account id is missing`, {webhookId: this.webhookId}) is a better pattern overall, adds the information to a more standard way

Reply:
Changed